src/main/java/org/folio/bulkops/service/ProfileService.java

```diff
@@ -91,13 +116,19 @@ public ProfileDto updateProfile(UUID profileId, ProfileUpdateRequest profileUpda
     if (isLocked || isLockAttempt) {
       validator.checkIfLockPermissionExists();
     }
+    Optional.ofNullable(profileUpdateRequest.getName()).ifPresent(existing::setName);
+    Optional.ofNullable(profileUpdateRequest.getDescription()).ifPresent(existing::setDescription);
+    Optional.ofNullable(profileUpdateRequest.getEntityType()).ifPresent(existing::setEntityType);
+    Optional.ofNullable(profileUpdateRequest.getRuleDetails()).ifPresent(existing::setRuleDetails);
```

Comment:
RuleDetails, MarcRuleDetails are the same as description. How can I clear these fields? I think profile isn't immutable so as a user I want to have editing possibility (including removing some parts of profile, i.e using nulls for these fields.).